frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/components/assisted-installer/DetailsForm.tsx

```diff
@@ -63,6 +64,11 @@ const DetailsForm: React.FC<DetailsFormProps> = ({ control, handleChange, contro
     const [managedClusterSet, setManagedClusterSet] = useState<string | undefined>()
     const [additionalLabels, setAdditionaLabels] = useState<Record<string, string> | undefined>({})
 
+    const getVersion = (versionName = '') => {
+        const clusterImage = clusterImageSets.filter((clusterImageSet) => clusterImageSet.metadata?.name == versionName)
+        return getVersionFromReleaseImage(clusterImage[0].spec?.releaseImage)
```

Comment:
just a bit more defensive approach (clusterImageSet can be deleted, in which case it would not be found and UI would crash)

```suggestion
        const clusterImage = clusterImageSets.find((clusterImageSet) => clusterImageSet.metadata?.name == versionName)
        return getVersionFromReleaseImage(clusterImage?.spec?.releaseImage) || versionName
```